protocol-correctness/lib/functions/last-functions.k

```diff
@@ -0,0 +1,46 @@
+require "protocol-correctness/lib/language/base-types.k"
+require "protocol-correctness/lib/language/reflection.k"
+require "protocol-correctness/lib/functions/plistlen-functions.k"
+
+module LAST-FUNCTIONS-SYNTAX
+  imports BASE-TYPES-SYNTAX
+
+  syntax Expression ::= last(ExpressionCSV)  [function]
+endmodule
+
+module LAST-FUNCTIONS-ADDITIONAL-SYNTAX
+  imports BASE-TYPES-SYNTAX
+
+  syntax Expression ::= #last(Expression, ExpressionCSV)  [function, functional]
+  syntax Expression ::= #last0(ExpressionCSV)  [function, functional]
```

Comment:
is `#last0` functional?

Reply:
I think we can choose option1 and have some comments in the code saying that it is intentional for debugging purpose.

Reply:
Done.